internal/payloads/reservation_payload.go

```diff
@@ -60,8 +60,8 @@ type AWSReservationResponsePayload struct {
 	// The ID of the image from which the instance is created.
 	ImageID string `json:"image_id"`
 
-	// The ID of the aws reservation which was created.
-	AWSReservationID string `json:"aws_reservation_id"`
+	// The ID of the aws reservation which was created, or missing if not created yet.
+	AWSReservationID string `json:"aws_reservation_id,omitempty"`
```

Comment:
Trying to omit it on empty. Not sure if that's more clear, tho. I'd love to be able to pass null